batchUploadResourceCollections.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This bash script is used to batch upload resource collections to the iHub server. It iterates through 
```

Comment:
I think this second sentence was started but not completed...

Reply:
Oops! Not sure what happened there.